client/widgets/CComponent.h

```diff
@@ -52,6 +52,7 @@ class CComponent : public virtual CIntObject
 	std::string customSubtitle;
 	ESize size; //component size.
 	EFonts font; //Font size of label
+	bool newLine; //Line break after component
```

Comment:
Why was this necessary?

Reply:
To make the spell (and only this one) in first line and ressources in following lines.